### DISCUSSION TITLE
Multiple Passwords for Inverter?

### DISCUSSION THREAD

**Discussion by judgeDirk** [SBFspot/SBFspot#533] [Q&A]
Hi There
I'm using SBFspot since 10/2020. No Problems so far. 
I'm logging 2 Inverters SMA SunnyBoy 5000TL (SB) and SMA Tripower 8.0 (TP) both connected by LAN.

Now my TP ask's me to change my Password  for the standard User. Stupidly i did it. So now i am facing a problem:
The SB and the TP have different passwords. With the standard pwd (0000) i receive from the SB only.
When i change the password in the config ... every thing is fine for the TP

- So i don't know how to change the password for the SB because there is no web-interface
- i can not set the password on the TP to 0000 because on password restrictions

is there a possible way to configure each system with an own pwd?
is there an way to change the password on the sunny boy without my electician?
any ideas?
thanks judge

**Comment by judgeDirk** [comment]
ok, solved it by myself
the missing piece was: SBFspot -cfgCONFIGFILE

1. splitted the config-file so that there are individual files for each inverter
2. changed the daily and the monthly skript that will started by cron
2a duplicated the log- and startup- entry
2b using the -cfg commandline parameter to pass the different configs

**Reply by sbf-** [reply]
With SunnyExplorer (windows only) you can change the password for the SB
The solution with 2 configfiles is OK but it makes it bit more complex
